apps/server/.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I added it to let everyone show how easily they can debug their code. However, this is generic. Whoever runs the<F5> button, can start debugging. I have seen many here writing console logs and wasting their time. They can increase their productivity and see the variables' value with it instead of console logging. 

However, this is a debatable topic whether it should be gitignored or not. 

By the way, it has a long debate in the internet. For example, I have read some pages from here: 

https://github.com/flutter/flutter/issues/15555

Lastly, will remove it.

Reply:
> what is this? this `NPM`? remove this file please

My mistake, it should be PNPM.